stacks/dovetail.prx.org.yml

```diff
@@ -350,72 +415,98 @@ Resources:
-        - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+      HealthCheckIntervalSeconds: 30
+      HealthCheckTimeoutSeconds: 10
+      HealthyThresholdCount: 2
+      UnhealthyThresholdCount: 2
+      HealthCheckPath: /
+      Name: !Sub dovetail-elixir-${EnvironmentTypeAbbreviation}-${VPC}
+      Port: 80
+      Protocol: HTTP
+      TargetGroupAttributes:
+        - Key: deregistration_delay.timeout_seconds
+          Value: "20"
       Tags:
         - Key: Project
-          Value: Infrastructure
+          Value: Dovetail
+        - Key: Environment
+          Value: !Ref EnvironmentType
+        - Key: Name
+          Value: !Sub DovetailElixir-${EnvironmentType}
         - Key: "prx:cloudformation:stack-name"
           Value: !Ref AWS::StackName
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
-  DovetailServiceDesiredCountRolloverLambdaFunction:
-    Type: "AWS::Lambda::Function"
+      VpcId: !Ref VPC
+  DovetailElixirService:
+    Type: "AWS::ECS::Service"
     Properties:
-      Code:
-        S3Bucket:
-          Fn::ImportValue:
-            !Sub ${InfrastructureStorageStackName}-InfrastructureSupportBucket
-        S3Key: utility/ecs-service-size-rollover.zip
-      Description: Allows for DesiredCount persistence on ECS services
-      Environment:
-        Variables:
-          DEFAULT_DESIRED_COUNT: "2"
-      Handler: index.handler
-      MemorySize: 128
-      Role: !GetAtt DovetailServiceDesiredCountRolloverIamRole.Arn
-      Runtime: nodejs8.10
+      Cluster: !Ref ECSCluster
+      DeploymentConfiguration:
+        MaximumPercent: 200
+        MinimumHealthyPercent: 50
+      DesiredCount: 2
+      LoadBalancers:
+        - ContainerName: "dovetail-elixir"
+          ContainerPort: 4000
+          TargetGroupArn: !Ref DovetailElixirALBTargetGroup
+      Role: !Ref ECSServiceIAMRole
       Tags:
         - Key: Project
-          Value: Infrastructure
+          Value: Dovetail
+        - Key: Environment
+          Value: !Ref EnvironmentType
         - Key: "prx:cloudformation:stack-name"
           Value: !Ref AWS::StackName
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
-      Timeout: 8
-  DovetailServiceDesiredCountRolloverResource:
-    Type: "Custom::EcsServiceSizeRollover"
-    Version: "1.0"
+      TaskDefinition: !Ref DovetailElixirTaskDefinition
+  DovetailElixirLogGroup:
+    Type: "AWS::Logs::LogGroup"
     Properties:
-      ClusterName: !Ref ECSCluster
-      Rotation: !Ref PipelineExecutionNonce
-      ServiceName: !Sub ${AWS::StackName}-DovetailApp
-      ServiceToken: !GetAtt DovetailServiceDesiredCountRolloverLambdaFunction.Arn
+      RetentionInDays: 14
+  DovetailElixirTaskDefinition:
+    Type: "AWS::ECS::TaskDefinition"
+    Properties:
+      ContainerDefinitions:
+        - Cpu: !Ref ContainerCpu
+          Environment:
+            - Name: APP_NAME
+              Value: "dovetail-elixir"
```

Comment:
I _think_ you can set the app-name as whatever you want in aws-secrets ... even including dashes.